person.rb

```diff
@@ -1,5 +1,5 @@
 require_relative('./corrector')
-
+require_relative('./rental')
 class Person < Nameable
   attr_accessor :name, :age
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/161922762-fdee4516-7db2-46ad-90dc-b0f0d27546a7.png)

Therefore:

- [ ] Please, kindly set the "has many" relationship "a person has many rentals".

## Hints

On line 4, you are just missing `:rental`:

```ruby
  attr_accessor :name, :age, :rental
```

That way, the relationship will be set 💪